distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -865,6 +868,13 @@ type Table
                     on_problems.handle_errors fallback=self.with_no_rows <|
                         mask (make_filter_column column filter)
                 _ : Function -> mask (column.map filter)
+        _ : Text ->
+            table_at = self.at column
+            if table_at.is_error.not then self.filter table_at filter on_problems else
+                expression = self.evaluate column
+                if expression.is_error.not then self.filter expression filter on_problems else
+                    pick_error = expression.catch Expression_Error.Syntax_Error (_->table_at)
+                    on_problems.handle_errors pick_error fallback=self
```

Comment:
Seems exactly the same as in DB, right? I'm wondering if we should extract it to some `Table_Helpers` to DRY. Not entirely sure though.